components/Slider/src/MDCSlider.h

```diff
@@ -367,6 +368,17 @@ IB_DESIGNABLE
  */
 @property(nonatomic, assign) BOOL shouldEnableHapticsForAllDiscreteValues;
 
+/**
+ The font of the discrete value label.
+
+ This font will come into effect only when @c numberOfDiscreteValues is larger than 0 and when @c
+ shouldDisplayDiscreteValueLabel is
+ @c YES.
+
+ Defaults to [[MDCTypography fontLoader] regularFontOfSize:12].
```

Comment:
Tip: MDCTypography is planned for deprecation in the near future.   Perhaps a comment that it is likely to change in the future as a result?

Reply:
done